src/components/Links.js

```diff
@@ -0,0 +1,12 @@
+import { Outlet, Link } from 'react-router-dom';
+
+const Links = () => (
+  <>
+    <div>
+      <Link to="/">Books</Link>
+      <Link to="categories">Categories</Link>
+    </div>
+    <Outlet />
```

Comment:
Hello @naolkidanu,

I've thoroughly compared your UI design with the reference design provided on Zeplin. Here are my observations and suggestions:

1. **Navbar Differences**:
   - **Typography & Color**: In your design, you've used the title "BooksCategories", whereas in the Zeplin design, the title reads "Bookstore CMS". It's essential to ensure that the typography aligns with the design specifications.
   - **Navbar Elements**: In the recommended design, we see a clear distinction between "BOOKS" and "CATEGORIES", separated by a small space in between. This distinction is missing in your design.
   - **User Icon**: The user icon on the top right in the Zeplin design is not present in your version. It's important to include all elements for consistency.

2. **Background Color**: The navbar background color in your design is different from the Zeplin design. Make sure to use the exact color codes provided in the design specifications.

3. **Consistency in Spacing**: There seems to be differences in spacing, especially in the navbar. Ensure you maintain consistent spacing and alignment as per the Zeplin design.


To ensure the best visual output and to match the expectations set by the design, it would be beneficial to adjust your design based on these points. Aligning closely with design specifications is a valuable skill in the development world, ensuring seamless collaboration with design teams.

Keep pushing forward! You're on the right path, and with a few tweaks, your design will be spot on. If you have any questions or need further clarification, please don't hesitate to ask.

Best wishes,
Temitope

<img width="935" alt="image" src="https://github.com/naolkidanu/bookstore/assets/52934128/3e21befd-1824-49c0-b106-c82b24c6187a">